tests/django_restframework_gis_tests/views.py

```diff
@@ -132,10 +132,12 @@ class GeojsonLocationNoIdDetails(generics.RetrieveUpdateDestroyAPIView):
 
 
 class LocationFilter(GeoFilterSet):
-    contains_properly = GeometryFilter(name='geometry', lookup_type='contains_properly')
```

Comment:
Are you sure removing ``lookup_type='contains_properly`` doesn't break compatibility with older django-filter versions?